static/src/javascripts/projects/common/modules/identity/modules/fetchFormFields.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+import fastdom from 'lib/fastdom-promise';
+
+export const ERR_IDENTITY_HTML_PREF_NOT_FOUND = `Can't find HTML preference`;
+
+export const getCsrfTokenFromElement = (
+    originalEl: ?HTMLElement
+): Promise<any> =>
```

Comment:
Could you be more specific with `any`? `any` basically disables are type checks and we are trying to discourage it and get rid of all occurences.